crates/holochain/src/conductor/conductor.rs

```diff
@@ -2156,29 +2156,63 @@ impl Conductor {
     /// (Cells belonging to Paused apps are not considered "dangling" and will not be removed)
     async fn remove_dangling_cells(&self) -> ConductorResult<()> {
         let state = self.get_state().await?;
-        let keepers: HashSet<CellId> = state
+
+        let keepers: HashSet<&CellId> = state
             .enabled_apps()
-            .flat_map(|(_, app)| app.all_cells().cloned().collect::<HashSet<_>>())
+            .flat_map(|(_, app)| app.all_cells().collect::<HashSet<_>>())
+            .collect();
+
+        let all: HashSet<&CellId> = state
```

Comment:
Is this really "all" there is?! =D

```suggestion
        let all_cells: HashSet<&CellId> = state
```